seed/migrations/0078_hpxml_building_file.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2017-11-28 23:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('seed', '0077_auto_20171024_1213'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='buildingfile',
+            name='file_type',
+            field=models.IntegerField(choices=[(0, 'Unknown'), (1, 'BuildingSync'), (2, 'GeoJSON'), (3, 'HPXML')], default=0),
```

Comment:
i will probably merge this migration into the larger bricr migration once I merge this down. just FYI